CONTRIBUTING.md

```diff
@@ -29,3 +29,23 @@ AMS uses a subset of [StandardIssueLabels](https://github.com/wagenet/StandardIs
 7. Create a new Pull Request
 
 Remember to squash your commits and rebase off `master`.
+
+### Running tests
```

Comment:
@bf4 Most users probably do not really care about actually running the tests I believe. What's your rationale behind adding it to README?